apex/contrib/sparsity/permutation_search_kernels/exhaustive_search.py

```diff
@@ -1,5 +1,8 @@
 from .permutation_utilities import *
 
+ASP_CACHE_DIR_ENV_VAR = "ASP_CACHE_DIR"
```

Comment:
How about `APEX_ASP_CACHE_DIR` to clarify the env var is for apex's asp module?

Reply:
Done.